htmresearch/support/csv_helper.py

```diff
@@ -198,10 +198,16 @@ def readDataAndReshuffle(args, categoriesInOrderOfInterest=None):
   # Read data
   dataDict = readCSV(args.dataPath, 1)
   labelRefs, dataDict = mapLabelRefs(dataDict)
+
+  if "numLabels" in args:
+    numLabels = args.numLabels
+  else:
+    numLabels = len(labelRefs)
```

Comment:
@subutai I've modified this method to not require you to specify numLabels in advance. If it's specified in args, that value will be used, but if not, it will use the number of unique labels as numLabels. Is this ok?